spec/support/authentication_helper.rb

```diff
@@ -11,5 +12,6 @@ def current_user
 
   def reset_authentication
     GDS::SSO.test_user = nil
+    Capybara.reset_sessions!
```

Comment:
It's a bit of a pain we need to add these as it couples this to Capybara. I guess it's necessary since warden must be not getting called if there is a user.

Reply:
Unfortunately, yes - seems to only work sometimes, otherwise.